spec/unit/core_spec.rb

```diff
@@ -123,4 +123,28 @@ class Book < ActiveFedora::Base
       it { should eq '123456w' }
     end
   end
+
+  describe "with an empty base_path" do 
+    let(:book) { Book.create(title:"foo") }
```

Comment:
@jcoyne I updated the tests to remove the duplicate code. I am torn between leaving them as they are now (validating the casting to the right class) vs validating the URL. I can go either way.